site-src/implementations.md

```diff
@@ -61,6 +63,22 @@ APISIX currently supports Gateway API `v1alpha2` version of the specification fo
 [apisix]:https://apisix.apache.org/
 [apisix-1]:https://github.com/apache/apisix-ingress-controller
 
+### Cilium
+
+[Cilium][cilium] is an eBPF-based networking, observability and security
+solution for Kubernetes and other networking environments. It includes [Cilium
+Service Mesh][cilium-service-mesh], a highly efficient mesh data plane that can
+be run in [sidecarless mode][cilium-sidecarless] to dramatically improve
+performance, and avoid the operational complexity of sidecars. Cilium also
+supports the sidecar proxy model, offering choice to users. Cilium is working on
+a Gateway API implementation.
```

Comment:
A link to any info specific to Gateway API would be great - a work-in-progress PR, or issue tracking the implementation, etc. I couldn't find anything with a quick search

Reply:
I agree, ideally we should have some kind of link here back to Cilium's implementation that viewers can follow.

Reply:
Added